Packages/Tracking/Examples~/XR Examples/Example Assets Old/Scripts/PullCord/PullCordHandle.cs

```diff
@@ -80,16 +87,29 @@ private void Update()
         {
             _handlePositionTarget = RestingPos;
 
-            if (_intBeh.isPrimaryHovered)
+            if ((_intBeh && _intBeh.isPrimaryHovered) || _relevanthands.Count() != 0)
```

Comment:
we should rename intBeh and any other reference to interaction engine to something more suitable - this is an example script, so is not breaking to change it if necessary